contributions/catalog/README.md

```diff
@@ -0,0 +1,60 @@
+# Contribution Catalog 
+
+### Prerequisites
+
+**MongoDB**
+
+- A system must have a mongodb installed.
+
+**[Python 3.5+](https://www.python.org)**
+
+
+## Set Up
+
+**MongoDB**
+ 
+ MongoDB should be installed
+
+**Configuration**
+
+The necessary configuration should be configured in configure file (configs.py) that is located under profileservice folder. This contains the MongoDB url, database name, collection name and so on. Modify the information in the file appropriately.
+
+## Environment File
+
+You need to have a `.env` file in this directory that contains credentials required for authentication. 
+Not all of these variables may be required for this building block.
+
+## Run application
+
+### Run locally without Docker
+
+This service uses the python Flask and PyMongo library.
+
+The configuration file configs.py should have the appropriate information
+
+To install and run the location-model service, do the following:
+
+## Setup Environment and start the catalog application
+- (Linux or MAC):
+```
+cd catalog
+virtualenv -p python3 venv
+source venv/bin/activate
+pip install -r requirements.txt
+set FLASK_APP=catalog
+set FLASK_ENV=development
+python contribution_catalog.py
+```
+- (Windows):
+```
+cd catalog
+py -m venv env
+.\env\Scripts\activate
+pip install -r requirements.txt
+set FLASK_APP=catalog
+set FLASK_ENV=development
+python contribution_catalog.py
+```
+
```

Comment:
Could you add how can the user get the actual end point? like http://localhost:5050?

Reply:
Also please add how to run using gunicorn and docker setting.